modules/spring-config/src/main/java/org/mule/runtime/config/internal/registry/CompositeOptionalObjectsController.java

```diff
@@ -8,13 +8,13 @@
 
 import static org.mule.runtime.api.util.Preconditions.checkArgument;
 
-import org.mule.runtime.core.internal.util.ArrayUtils;
-
 import java.util.Collection;
 import java.util.List;
 
 import com.google.common.collect.ImmutableList;
 
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
Is this intentional?

Reply:
yes, this replaces `org.mule.runtime.core.internal.util.ArrayUtils` (removed up there, line 11)